sns/parity_math.h

```diff
@@ -43,6 +43,12 @@
    @{
 */
 
+/**
+ *  Define the condition here to use Intel ISA library.
+ */
+#define ISAL_ENCODE_ENABLED	(!defined(__KERNEL__))
```

Comment:
Once [PR-416](https://github.com/Seagate/cortx-motr/pull/416) is merged, this can be updated to 
`!defined(__KERNEL__) && defined(HAVE_ISAL)`